build/common.sh

```diff
@@ -58,69 +100,71 @@ function _log() {
-        \"git.remote.origin.url\": ${git_labels_git_remote_origin_url} \
-      },                                                               \
-      \"message\": ${message}                                          \
-      }"                                                               \
-    ) || { _feedback ERROR "Failed to generate a valid JSON log message"; _log "easy_infra.stdouterr" info unknown "easy_infra" "${label_cwd}" string "Failed to generate a valid JSON log message"; }
+    log_header="${log_header},                                       \
+      \"git.branch.name\": ${git_labels_git_branch_name},            \
+      \"git.branch.ref\": ${git_labels_git_branch_ref},              \
+      \"git.remote.origin.url\": ${git_labels_git_remote_origin_url} \
+    },"
+  else
+    log_header="${log_header}},"
+  fi
+
+  log_header="${log_header}\"message\":"
+  local log_trailer
+  log_trailer="}"
+
+  local one_line_log
+  one_line_log="${log_header} ${message} ${log_trailer}"
+
+  # This adds a newline to the end of the one line log and write it out
+  echo "${one_line_log}" > "${temporary_log}"
+
+  # Validate the log using jq
+  if jq empty "${temporary_log}" &>/dev/null ; then
+    # Ensure that the temporary log is all on one line
+    jq -c < "${temporary_log}" &>> /var/log/easy_infra.log
   else
-    LOG_MESSAGE=$(jq -c -n                               \
-      "{                                                 \
-      \"@timestamp\": ${timestamp},                      \
-      \"container.image.name\": ${container_image_name}, \
-      \"container.image.tag\": ${container_image_tag},   \
-      \"ecs.version\": ${ecs_version},                   \
-      \"event.action\": ${event_action},                 \
-      \"event.category\": ${event_category},             \
-      \"event.kind\": ${event_kind},                     \
-      \"event.outcome\": ${event_outcome},               \
-      \"event.provider\": ${event_provider},             \
-      \"event.type\": ${event_type},                     \
-      \"labels\": {                                      \
-        \"cwd\": ${label_cwd}                            \
-      },                                                 \
-      \"message\": ${message}                            \
-      }"                                                 \
-    ) || { _feedback ERROR "Failed to generate a valid JSON log message"; _log "easy_infra.stdouterr" info unknown "easy_infra" "${label_cwd}" string "Failed to generate a valid JSON log message"; }
+    _feedback ERROR "Failed to generate a valid JSON log message"
+    _log "easy_infra.stdouterr" info unknown "easy_infra" "${label_cwd}" string "Failed to generate a valid JSON log message"
   fi
-  echo "${LOG_MESSAGE}" &>> /var/log/easy_infra.log
 }
 
 
 function _feedback() {
-  timestamp="$(date -Ins)"
+  local timestamp
+  timestamp="$(date --iso-8601=seconds --utc)"
```

Comment:
This makes the timestamps consistent between feedback and logs, and also shortens the feedback (what goes to stdout/err)